www/src/components/api-reference/doc-block.js

```diff
@@ -0,0 +1,134 @@
+import React from "react"
+import styled from "react-emotion"
+
+import ExamplesBlock from "./examples"
+import ParamsBlock from "./params"
+import {
+  SignatureWrapper,
+  TypeComponent,
+  SignatureElement,
+  isFunctionDef,
+} from "./signature"
+import ReturnBlock from "./returns"
+import { Header } from "./utils"
+import { options, scale } from "../../utils/typography"
+
+const Optional = styled.span`
```

Comment:
unsure if screen readers read the content field. ping @marcysutton

Reply:
That's good point. It probably makes sense to adjust. I'm just not sure what semantic structure actually makes sense to use here.

Reply:
Yes they do read it! It becomes part of the computed text for an element node.